README.md

```diff
@@ -55,36 +58,38 @@ logzio/fluentd-docker-logs
+| LOGZIO_PROXY_CERT               | **Default**: `nil` <br> Your proxy cert.                                                                                                                                                   |
+| ADDITIONAL_FIELDS               | Include additional fields with every message sent, formatted as "fieldName1=fieldValue1,fieldName2=fieldValue2".                                                                           |
+| LOGZIO_CONTAINER_STATUS_REGEX   | **Default**: `running` <br> Regex to match against container status. One or more of `created`, `restarting`, `running`, `removing`, `paused`, `exited`, or `dead`.                         |
+| LOGZIO_SLOW_FLUSH_LOG_THRESHOLD | **Default**: `20.0` <br> The threshold for chunk flush performance check.                                                                                                                  |
+| LOGZIO_BUFFER_TYPE              | **Default**: `file` <br> Specifies which plugin to use as the backend.                                                                                                                     |
+| LOGZIO_BUFFER_PATH              | **Default**: `/var/log/Fluentd-buffers/stackdriver.buffer` <br> Path of the buffer.                                                                                                        |
+| LOGZIO_OVERFLOW_ACTION          | **Default**: `block` <br> Controls the behavior when the queue becomes full. Refer to [fluentd docs](https://docs.fluentd.org/output#overflow_action) for more details about your options. |
+| LOGZIO_CHUNK_LIMIT_SIZE         | **Default**: `2M` <br> Maximum size of a chunk allowed                                                                                                                                     |
+| LOGZIO_QUEUE_LIMIT_LENGTH       | **Default**: `6` <br> Maximum length of the output queue.                                                                                                                                  |
+| LOGZIO_FLUSH_INTERVAL           | **Default**: `5s` <br> Interval, in seconds, to wait before invoking the next buffer flush.                                                                                                |
+| LOGZIO_RETRY_MAX_INTERVAL       | **Default**: `30s` <br> Maximum interval, in seconds, to wait between retries.                                                                                                             |
+| LOGZIO_FLUSH_THREAD_COUNT       | **Default**: `2` <br> Number of threads to flush the buffer.                                                                                                                               |
+| LOGZIO_LOG_LEVEL                | **Default**: `info` <br> The log level for this container.                                                                                                                                 |
```

Comment:
Did you add the new env var to the table?

Reply:
yes